core/src/main/scala/org/apache/spark/deploy/DeployMessage.scala

```diff
@@ -188,7 +188,7 @@ private[deploy] object DeployMessages {
   }
 
   case class ExecutorUpdated(id: Int, state: ExecutorState, message: Option[String],
-    exitStatus: Option[Int], workerLost: Boolean)
+    exitStatus: Option[Int], hostOpt: Option[String])
```

Comment:
I think we need a better name than hostOpt ? How about just "Hostname" ? The type already conveys that this is an optional.

Reply:
How about hostLost ?

Reply:
After a second thought, I changed it to `workerHost`. We need the keyword `worker` because it's specific to Standalone Worker. And `Host` gives the direct meaning of the value. And `workerLost` sounds more appropriate for the `Boolean` type. WDYT?